presto-main/src/main/java/io/prestosql/sql/planner/plan/JoinNode.java

```diff
@@ -133,6 +134,10 @@ public JoinNode(
 
     public JoinNode flipChildren()
     {
+        Map<String, Symbol> validDynamicFilters = dynamicFilters.entrySet().stream()
```

Comment:
I don't think there can be any valid dynamic filters at all after flipping children, can they?

Reply:
I was initially thinking of the case where current join node is on the probe side of some join node above it, I realize now that the current join node's dynamic filters map won't contain the downstream join node's dynamic filters, only the filter nodes below it contain the relevant predicates which are unaffected by the flip.
I'll fix this to send empty map for dynamic filters.